src/main/resources/logback-base.xml

```diff
@@ -1,12 +1,38 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE xml>
-<configuration scan="true" scanPeriod="30 seconds" >
+<configuration>
     <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
-        <encoder class="net.logstash.logback.encoder.LogstashEncoder"/>
+        <encoder>
```

Comment:
Per il ConsoleAppender dovremmo mantenere il LogstashEncoder per loggare in JSON in CloudWatch mi sa...